lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_methods.rb

```diff
@@ -7,12 +7,17 @@ class MiqAeServiceMethods
 
     SYNCHRONOUS = Rails.env.test?
 
-    def self.send_email(to, from, subject, body, content_type = nil)
+    def self.send_email(to, from, subject, body, args = {})
```

Comment:
This method is being used only in automate. I don't see any caller of the method that specifies ```content_type```.

The  method signature can be changed to:
```
def self.send_mail(to, from, subject, body, content_type: nil, cc: nil, bcc: nil)
```